dbt_copilot_helper/COMMANDS.md

```diff
@@ -706,6 +710,72 @@ copilot-helper domain assign --app <app> --env <env> --svc <svc>
 - `--help <boolean>` _Defaults to False._
   - Show this message and exit.
 
+# copilot-helper environment
+
+[↩ Parent](#copilot-helper)
+
+    Commands affecting environments.
+
+## Usage
+
+```
+copilot-helper environment (offline|online) 
+```
+
+## Options
+
+- `--help <boolean>` _Defaults to False._
+  - Show this message and exit.
+
+## Commands
+
+- [`offline` ↪](#copilot-helper-environment-offline)
+- [`online` ↪](#copilot-helper-environment-online)
+
+# copilot-helper environment offline
+
+[↩ Parent](#copilot-helper-environment)
+
+    Take load-balanced web services offline with a maintenance page.
+
+## Usage
+
+```
+copilot-helper environment offline --app <app> --env <env> [--template (default|migration)] 
+```
+
+## Options
+
+- `--app <text>`
+
+- `--env <text>`
+
+- `--template <choice>` _Defaults to default._
```

Comment:
`Defaults to default` doesn't read well, but I am not sure if we can do anything about that.

Reply:
Other than change `default` to something else, but I think this makes sense.